git_machete/cmd.py

```diff
@@ -3482,6 +3457,7 @@ def main() -> None:
 def launch(orig_args: List[str]) -> None:
 
     cli_ctxt = CommandLineContext()
```

Comment:
Hmmm `CommandLineContext` is created here... but isn't it also created inside of `MacheteContext` as a static field? 🤔 I don't think we should have 2 distinct instances of `CommandLineContext` in the application

Reply:
I agree, at first step i wanted to stick to proposal in PR #124. In my opinion `CommandLineContext` object should be passed to `MacheteContext` constructor or created there if it's not passed.